index.bs

```diff
@@ -2279,7 +2279,8 @@ Registering Formal Objections</h3>
 	rationales,
 	and decisions <em class="rfc2119">should</em> also be recorded.
 
-	A record of each [=Formal Objection=] against a decision regarding a publicly-available document
+	Upon registration with the [=Team=],
```

Comment:
I guess this implies that registration needs to include the exact wording that'll become public, so the Team doesn't need to spend time going through [7.3. Changing Confidentiality Level](https://www.w3.org/2023/Process-20231103/#confidentiality-change). That's fine with me, but a delay of up to, say, a week, would also be fine with me.

Reply:
The intention was to give some time to the Team to do that, so if this seems a bit too instantaneous, we should pick some other wording here. :)

Reply:
Based on the 2023-12-13 Process CG call:
```suggestion
	No later than when the <a href="#team-fo-mediation">relevant council is initiated<a>,
```

Reply:
(More precise guidance will be written up as a /Guide article, with an Action on @plehegar to draft that.)